concordium-node/src/consensus_ffi/helpers.rs

```diff
@@ -180,30 +181,70 @@ impl PacketType {
     }
 }
 
-#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+#[derive(Debug, PartialEq, Eq, Clone, Copy, Error)]
 pub enum ConsensusFfiResponse {
+    #[error("Baker not found")]
     BakerNotFound = -1,
+    #[error("Success")]
     Success,
+    #[error("Deserialization error")]
     DeserializationError,
+    #[error("Invalid result")]
     InvalidResult,
+    #[error("Pending block")]
     PendingBlock,
+    #[error("Pending finalization")]
     PendingFinalization,
+    #[error("Asynchronous")]
     Asynchronous,
+    #[error("Duplicate entry")]
     DuplicateEntry,
+    #[error("Stale")]
     Stale,
+    #[error("Incorrect finalization session")]
     IncorrectFinalizationSession,
+    #[error("Unverifiable")]
     Unverifiable,
+    #[error("Continue catchup")]
     ContinueCatchUp,
+    #[error("Block too early")]
     BlockTooEarly,
+    #[error("Missing import file")]
     MissingImportFile,
+    #[error("Consensus shutdown")]
     ConsensusShutDown,
+    #[error("Expiry too late")]
     ExpiryTooLate,
+    #[error("Verification failed")]
     VerificationFailed,
+    #[error("Nonexisting sender account")]
     NonexistingSenderAccount,
+    #[error("Duplicate nonce")]
     DuplicateNonce,
+    #[error("Nonce too large")]
     NonceTooLarge,
+    #[error("Too low energy")]
     TooLowEnergy,
+    #[error("Invalid genesis index")]
     InvalidGenesisIndex,
+    #[error("An account already exists with the given registration id")]
+    DuplicateAccountRegistrationID,
+    #[error("The credential deployment contained invalid signatures")]
+    CredentialDeploymentInvalidSignatures,
+    #[error("The credential deployment contained an invalid identity provider")]
+    CredentialDeploymentInvalidIP,
+    #[error("The credential deployment contained invalid anonymity revokers")]
+    CredentialDeploymentInvalidAR,
+    #[error("The credential deployment was expired")]
+    CredentialDeploymentExpired,
+    #[error("The chain update had an invalid effective time")]
+    ChainUpdateInvalidEffectiveTime,
+    #[error("The chain update had an old nonce")]
+    ChainUpdateSequenceNumberTooOld,
+    #[error("The chain update contained invalid signatures")]
+    ChainUpdateInvalidSignatures,
+    #[error("The stated energy of the transaction exceeds the maximum allowed.")]
```

Comment:
Some of these errors end with a `.`, but some do not, even if they otherwise read as full sentences. This should be fixed one way or another.